Main.java

```diff
@@ -1,15 +1,30 @@
-/**
- *
- * @author 
- */
-public class Main {
-
-  /**
-   * The method that is executed when you hit the run button.
-   * @param args the command line arguments
-   */
-  public static void main(String[] args) {
-    // TODO code application logic here
-    
-  }
-}
+import java.util.Scanner; 
+
+/**
+ *To convert Celcius into Fahrenheit
+ * @author Zach Ramirez 
+ */
+public class Main {
+
+  /**
+   * The method that is executed when you hit the run button.
+   * @param args the command line arguments
+   */
+  public static void main(String[] args) {
+    // create the scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    String temperature;
+
+    // ask user for a temperature
+    System.out.println("Please enter your temperature in celcius:");
+    //initialize the users temperature
+    int temp = input.nextInt();
+
+    final int CELCIUS_FAHRENHEIT = temp * 9 / 5 + 32;
+
+    int finalTemp = CELCIUS_FAHRENHEIT;
+
+    System.out.println("your temperature is " + finalTemp);
```

Comment:
Good work. I'm not really sure you need the constant here but it is ok since it doesn't change. I would also like to see some comment about what is happening throughout. Think of jotting down the steps of how to solve the problem and using that as your comments. So we should see a comment about the math happening and also on the output happening. Finally, the reason you don't pass the test cases is because it doesn't match the sample output. Try to stay as close to the sample output as you can. The testing depends on it matching EXACTLY